requirements/base.txt

```diff
@@ -0,0 +1,2 @@
+aiofiles>=0.5,<=23.2.1
+aiohttp>=3.8.1,<4
```

Comment:
We'll need click here as well for the CLI stuff.

Reply:
oh yeah! will add.